### ISSUE TITLE
key groups

### ISSUE THREAD

**Issue by dorseth** [leo-buneev/eslint-plugin-sort-keys-fix#4]
It would be nice if one could specify order of some keys, and sort those, then sort others alpabetically.

**Comment by dimitropoulos** [comment]
I agree.  The use case for this I have in mind is when you are doing css-in-js and you want to be able to keep related css properties together.

Some prior art here that's similar but different: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md

**Comment by chancegraff** [comment]
Old issue but I'd keep using this plugin if this were implemented.

**Comment by entropitor** [comment]
One way to do this would be to do it based on a newline, so if there is a newline, sort all the partitions but don't move stuff over newlines (this could be an option)
